test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+Help()
+{
+	echo "Syntax: ./test.sh [-f filename] [-i]"
+	echo "options:"
+	echo "f	    Insert argument for file name (gfa)"
+	echo "i     Creates folder for intermediate files called test_files, instead of deleting them"
+	echo "h     Displays Help"
+}
+
+while getopts 'f:ih' OPTION; do
+	case "$OPTION" in
+		f)
+			odgi build -g $OPTARG -o temp.og
+			odgi depth -i temp.og -d > temp_depth.txt
+
+			python3 process.py temp_depth.txt
+
+			python3 depth.py $OPTARG
```

Comment:
If you take my suggestion above to avoid hard-coding the output filename, you can do `> python_output.txt` here.